src/index.js

```diff
@@ -0,0 +1,64 @@
+import './styles/style.css';
+
+import { addLike, getLikes } from './modules/like.js';
+import { fetchAllMovies } from './modules/movieApi.js';
+import { countMovies } from './modules/Counters.js';
+
+const renderMovies = (movies, likes) => {
+  const moviesContainer = document.querySelector('.movies-container');
+  movies.forEach(
+    ({
+      id, name, image, language, description,
+    }, index) => {
+      const movieCard = `
+        <div class="container" id=${id}>
+          <img src=${image.medium}>
+          <span>${name}</span>
+          <span>Language: ${language}</span>
+          <span> Description: ${description}</span>
```

Comment:
- [ ] Impressive work for building such a beautiful UI. Please note that the `description` of the movies is returning `undefined`. Kindly consider fixing this issue to help your users to have a brief description of the movies you have displayed as it will help create a better user experience. See below for more details:
<img width="394" alt="Screenshot 2023-06-10 at 16 42 32" src="https://github.com/hagan199/JavascriptCapstone/assets/107192076/89f27ed4-c1aa-416e-ba2a-71b10f20b996">